translations/kubectl/pt_BR/LC_MESSAGES/k8s.po

```diff
@@ -0,0 +1,3319 @@
+"\t\tkubectl get -o template pod/web-pod-13je7 --template={{.status.phase}}\n"
+"\n"
+"\t\t# List all replication controllers and services together in ps output "
+"format.\n"
+"\t\tkubectl get rc,services\n"
+"\n"
+"\t\t# List one or more resources by their type and names.\n"
+"\t\tkubectl get rc/web service/frontend pods/web-pod-13je7\n"
+"\n"
+"\t\t# List all resources with different types.\n"
+"\t\tkubectl get all"
+msgstr ""
+"\n"
+"\t\t# Lista todos os pods no formato de saída ps.\n"
+"\t\tkubectl get pods\n"
+"\n"
+"\t\t# Lista todos os pods no formato de saída ps com mais informações (como "
+"o nome do node).\n"
+"\t\tkubectl get pods -o wide\n"
+"\n"
+"\t\t# Lista um único replication controller com o NAME especificado no "
+"formato de saída ps\n"
+"\t\tkubectl get replicationcontroller web\n"
+"\n"
+"\t\t# Lista um único pod e usa o formato de saída JSON.\n"
+"\t\tkubectl get -o json pod web-pod-13je7\n"
+"\n"
+"\t\t# Lista o pod identificado com o tipo e nome especificado no “pod.yaml” "
+"e usa o formato de saída JSON.\n"
+"\t\tkubectl get -f pod.yaml -o json\n"
+"\n"
+"\t\t# Mostra apenas em que estágio o pod especificado está.\n"
+"\t\tkubectl get -o template pod/web-pod-13je7 —template={{.status.phase}}\n"
+"\n"
+"\t\t# Lista todos os replication controllers e services juntos no formato de "
+"saída ps.\n"
+"\t\tkubectl get rc,services\n"
+"\n"
+"\t\t# Lista um ou mais recursos pelo seu tipo e nomes.\n"
+"\t\tkubectl get rc/web service/frontend pods/web-pod-13je7\n"
+"\n"
+"\t\t# Lista todos os recursos e com tipos diferentes.\n"
+"\t\tkubectl get all"
+
+#: pkg/kubectl/cmd/portforward.go:53
+msgid ""
+"\n"
+"\t\t# Listen on ports 5000 and 6000 locally, forwarding data to/from ports "
+"5000 and 6000 in the pod\n"
+"\t\tkubectl port-forward mypod 5000 6000\n"
+"\n"
+"\t\t# Listen on port 8888 locally, forwarding to 5000 in the pod\n"
+"\t\tkubectl port-forward mypod 8888:5000\n"
+"\n"
+"\t\t# Listen on a random port locally, forwarding to 5000 in the pod\n"
+"\t\tkubectl port-forward mypod :5000\n"
+"\n"
+"\t\t# Listen on a random port locally, forwarding to 5000 in the pod\n"
+"\t\tkubectl port-forward mypod 0:5000"
+msgstr ""
+"\n"
+"\t\t# Escuta nas portas locais 5000 e 6000, e redireciona os dados de/para "
+"as portas 5000 e 6000 no pod\n"
+"\t\tkubectl port-forward mypod 5000 6000\n"
+"\n"
+"\t\t# Escuta na porta local 8888 locally, e redireciona para a porta 5000 no "
+"pod\n"
+"\t\tkubectl port-forward mypod 8888:5000\n"
+"\n"
+"\t\t# Escuta uma porta local aleatória, e redireciona para a porta 5000 no "
+"pod\n"
+"\t\tkubectl port-forward mypod :5000\n"
+"\n"
+"\t\t# Escuta uma porta local aleatória, e redireciona para a porta 5000 no "
+"pod\\n\n"
+"\t\tkubectl port-forward mypod 0:5000"
+
+#: pkg/kubectl/cmd/drain.go:118
+msgid ""
+"\n"
+"\t\t# Mark node \"foo\" as schedulable.\n"
+"\t\t$ kubectl uncordon foo"
+msgstr ""
+"\n"
+"\t\t# Marca o node “foo” como agendável.\n"
```

Comment:
I think here, maybe we need a less literal translation. Something like "Remove a restrição de execução de Pods no node 'foo'.